src/app/app.component.html

```diff
@@ -14,7 +14,7 @@ <h2><a target="_blank" rel="noopener" href="https://angular.io/tutorial">Tour of
     <h2><a target="_blank" rel="noopener" href="https://angular.io/cli">CLI Documentation</a></h2>
   </li>
   <li>
-    <h2><a target="_blank" rel="noopener" href="https://blog.angular.io/">Angular blog</a></h2>
+    <h2><a href="/profile">Player profile</a></h2>
```

Comment:
use routerLink directive to navigate between routes
https://angular.io/api/router/RouterLink